src/llm/llm_calculator.cc

```diff
@@ -0,0 +1,87 @@
+//*****************************************************************************
+// Copyright 2024 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "mediapipe/framework/calculator_framework.h"
+#pragma GCC diagnostic pop
+
+#include <continuous_batching_pipeline.hpp>
+#include <generation_config.hpp>
+#include <memory>
+#include <openvino/openvino.hpp>
+#include <scheduler_config.hpp>
+#include <tokenizer.hpp>
```

Comment:
where is it coming from?

Reply:
From openvino.genai repo -> llm_engine cmake -> text_generation/causal_lm/cpp/continuous_batching/library/**